### ISSUE TITLE
4.9.1 artifact is broken

### ISSUE THREAD

**Issue by larsrh** [etorreborre/specs2#810]
Same issue as typelevel/cats#3313. This is caused by an "unclean" release.

```
[error] (coreJS / Test / fastOptJS) java.io.IOException: Failed to deserialize /home/travis/.cache/coursier/v1/https/repo1.maven.org/maven2/org/specs2/specs2-common_sjs1_2.12/4.9.1/specs2-common_sjs1_2.12-4.9.1.jar:/org/specs2/control/producer/Producers$TransducerOps.sjsir
```

**Comment by etorreborre** [comment]
Ok, thanks for the heads-up Lars, I'm going to redo a clean one then.

**Comment by etorreborre** [comment]
Done I've now published `specs2-4.9.2`. Thanks again!
